### PR TITLE
Feature/update prisma

### PR DESCRIPTION
**Pull request by benistary**
- replace prismjs with prism-react-renderer
- change Code.js component from class-based to functional
- update and fix styling

### CODE REVIEW

File: packages/styleguide/package.json [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
     "polished": "~2.3.1",
     "pretty": "~2.0.0",
+    "prism-react-renderer": "~1.0.2",
     "prism-theme-one-dark": "~1.0.0",
-    "prismjs": "~1.15.0",
     "promise": "^8.0.3",
     "react": "~16.10.1",
     "react-dom": "~16.10.1",
-    "react-element-to-jsx-string": "~14.1.0",
+    "react-docgen": "~4.1.0",
```
Comment (adammockor) [inline packages/styleguide/package.json]: Toto sa sem ako dostalo?

File: packages/styleguide/src/components/Code/Code.js [hunk 2]
```diff
@@ -1,124 +1,103 @@
...
+            {line.map((token, key) => (
+              <span {...getTokenProps({ token, key })} />
+            ))}
+          </Tag>
+        ));
+
+        return (
+          <StyledHighlightWrapper inline={inline} as={inline ? 'span' : 'div'}>
+            {inline ? (
+              <code
```
Comment (adammockor) [inline packages/styleguide/src/components/Code/Code.js]: A nemal by byt `<code />` aj v `<pre />`?

File: packages/styleguide/package.json (hunk 1 above)
Comment (benistary) [reply packages/styleguide/package.json]: riesil som merge conflict tak asi vtedy 💁‍♂ 😄 dam prec

File: packages/styleguide/src/components/Code/Code.js (hunk 2 above)
Comment (benistary) [reply packages/styleguide/src/components/Code/Code.js]: fixnute 👍

**Pull request merged**